exporters/otlp/internal/transform/attribute.go

```diff
@@ -74,10 +74,170 @@ func toAttribute(v kv.KeyValue) *commonpb.KeyValue {
 		result.Value.Value = &commonpb.AnyValue_StringValue{
 			StringValue: v.Value.AsString(),
 		}
+	case kv.ARRAY:
+		result.Value.Value = toArrayAttribute(v)
 	default:
 		result.Value.Value = &commonpb.AnyValue_StringValue{
 			StringValue: "INVALID",
 		}
 	}
 	return result
 }
+
+func toArrayAttribute(v kv.KeyValue) *commonpb.AnyValue_ArrayValue {
+	array := v.Value.AsArray()
+	resultValues := []*commonpb.AnyValue{
+		{
+			Value: &commonpb.AnyValue_StringValue{
+				StringValue: "INVALID",
+			},
+		},
+	}
+	if boolArray, ok := array.([]bool); ok {
```

Comment:
A [type switch](https://golang.org/ref/spec#TypeSwitchStmt) would cut down on some of the boilerplate here

Reply:
True. Changed it.